tests/lit-tests/1576.ispc

```diff
@@ -0,0 +1,5 @@
+// RUN: %{ispc} --target=avx2-i32x16,avx512skx-i32x16 %s > %t 2>&1
```

Comment:
Is this going to work without FileCheck? Do we need FileCheck is we just checking that compiler is not crashing (returning 0)?

Reply:
The test will fail even without the FileCheck since it crashes but I've added it and squashed the 2 separate commits.